topics/dev/images/Makefile

```diff
@@ -1,9 +1,11 @@
+MINDMAPS := $(wildcard *.mindmap.yml)
 INPUTS := $(wildcard *.plantuml.txt)
 OUTPUTS := $(INPUTS:.txt=.svg)
 
-all: plantuml.jar plantuml_options.txt $(OUTPUTS)
+all: plantuml.jar $(MINDMAPS) $(OUTPUTS)
 
 $(OUTPUTS): $(INPUTS)
```

Comment:
```suggestion
$(OUTPUTS): $(INPUTS) $(MINDMAPS)
```

Reply:
Put this in the latest rebase, thanks for the suggestion!